test/integration/multiplexed_integration_test.cc

```diff
@@ -540,6 +554,9 @@ TEST_P(Http2MetadataIntegrationTest, TestResponseMetadata) {
   ASSERT_TRUE(response->waitForEndStream());
   ASSERT_TRUE(response->complete());
   std::set<std::string> expected_metadata_keys = {"headers", "duplicate"};
+  if (upstreamProtocol() == Http::CodecType::HTTP3) {
+    expected_metadata_keys.insert("data");
```

Comment:
comment at least the first of these?  I assume this is one of those "H3 sending zero byte chunk" differences?

Reply:
Done. (Commented all of the places to be consistent.)